app/controllers/requests_controller.rb

```diff
@@ -0,0 +1,21 @@
+class RequestsController < ApplicationController
+
+  def index
+    @requests = current_user.requests
+  end
+
+  def Show
```

Comment:
Delete this method if it isn't used.

Also remember: method names are always in lower case.